cmd/thanos/query.go

```diff
@@ -65,7 +66,7 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application, name string
 	selectorLabels := cmd.Flag("selector-label", "Query selector labels that will be exposed in info endpoint (repeated).").
 		PlaceHolder("<name>=\"<value>\"").Strings()
 
-	stores := cmd.Flag("store", "Addresses of statically configured store API servers (repeatable).").
+	stores := cmd.Flag("store", "Addresses of statically configured store API servers (repeatable). The scheme may be prefixed with 'dns+' or 'dnssrv+' to detect store API servers through respective DNS lookups.").
 		PlaceHolder("<store>").Strings()
 
 	fileSDFiles := cmd.Flag("store.sd-files", "Path to files that contain addresses of store API servers. The path can be a glob pattern (repeatable).").
```

Comment:
Just spotted, but I think we need singular form for consistency.

Reply:
But here you can pass multiple files using globbing, while in the case of the static flags you can only pass one (repeatedly). What do you think?